CMakeLists.txt

```
@@ -40,7 +43,7 @@ if(YSTDLIB_CPP_IS_TOP_LEVEL)
     # Include dependency settings if the project isn't being included as a subproject.
     # NOTE: We mark the file optional because if the user happens to have the dependencies
     # installed, this file is not necessary.
-    include(build/deps/settings.cmake OPTIONAL)
+    include(build/deps/CMake/settings.cmake OPTIONAL)
```

nit: add quotes to paths.

```suggestion
    include("build/deps/CMake/settings.cmake" OPTIONAL)
```